mwxml/iteration/revision.py

```diff
@@ -49,6 +50,8 @@ def from_element(cls, element):
                     user = User.from_element(sub_element)
             elif tag == "minor":
                 minor = True
+            elif tag == "origin":
+                origin = sub_element.text
```

Comment:
We'll need to add "origin" to the mwtypes.Revision model too and make sure it gets set.